### ISSUE TITLE
Jest = Can not find module

### ISSUE THREAD

**Issue by remijean** [adamgruber/sass-extract-js#12]
Hello,

I have a problem with Jest, when I run a test I have this error:

Cannot find module 'sass-extract-loader?{"plugins": ["sass-extract-js"]}!./theme/variables.scss' from 'index.tsx'

My code : 

```
const theme = require('sass-extract-loader?{"plugins": ["sass-extract-js"]}!./theme/variables.scss')

ReactDOM.render(
  <ThemeProvider theme={theme}>
    ...
  </ThemeProvider>
)
```

**Comment by adamgruber** [comment]
🤔 This may be an issue with [sass-extract-loader](https://github.com/jgranstrom/sass-extract-loader/) rather than this plugin.

**Comment by rcwestlake** [comment]
@remijean I'm seeing the same thing. Did you resolve it?

**Comment by remijean** [comment]
@rcwestlake no, I have not found a solution, but I mentioned the issue here https://github.com/jgranstrom/sass-extract-loader/issues/7.

**Comment by adi-works** [comment]
No Jest support is a deal-breaker and the workaround posted in jgranstrom/sass-extract-loader#7 isn't good enough.

**Comment by adamgruber** [comment]
Unfortunately this is a [limitation with jest-resolver](https://github.com/facebook/jest/issues/4868) not understanding the webpack import syntax.

You could try mocking the loader similar to how it's done in [this issue](https://github.com/webpack-contrib/imports-loader/issues/54).
